app/services/notification_service.rb

```diff
@@ -38,6 +38,9 @@ def self.send_for_follows(follows, entity, event_type, event)
         when TOPIC_ADDED_TO_MAP
           next unless TopicPolicy.new(follow.user, entity).show? && MapPolicy.new(follow.user, event.map).show?
           next if follow.user.has_map_open(event.map)
+          # note, the caveat to this fork related code is that no future updates will be sent
+          # either. only the notifications for the initial fork should be squashed like this
+          next if event.map.source_id # if map is a fork, send notification for that instead
```

Comment:
I guess the options are:

1. store a field on a map that says "I'm a fork, and I have/have not already copied all the topics from my parent map". This could be unstable if there are errors during the forking though.
2. something else more complicated as part of a larger fork logic revamp
3. just check the created_at time here and don't send notifications for the first 30 seconds (this is suuuuper hacky and could lead to stuff being missed right after an initial fork, but since no one should be following the fork at first anyways, maybe it's ok??)

(2) seems best haha, although (3) is intriguing

Reply:
lol, I was thinking about doing number 3, but then opted not to, in the hopes that we'll be incentivized to do something more like 2 sooner :)